### ISSUE TITLE
Option '--qf' applies only to parameter attached to option '-q', not to each output row

### ISSUE THREAD

**Issue by Ricky-Tigg** [rpm-software-management/rpm#600]
Command option '_--qf_' applies **only** to **parameter attached** to option '-_q_'  (here _rpm_), not to each output row as expected. (Possible bug)

Actual result:
```
$ rpm -qR rpm --qf '%{NAME} – %{DESCRIPTION}'
/usr/bin/bash
(...)
rpm – The RPM Package Manager (RPM) (...)
(...) a description, etc. [_@l_~]$
``` 
Expected result: specified '--qf' **fields** (here _NAME_, _DESCRIPTION_) to be applied to **each** row of an output resulting from '_rpm -qR rpm_'.

**Comment by Ricky-Tigg** [comment]
Indeed. I had figured out the supported fields list using ’rpm --querytags’. Yet I fail to discover a way to print each tag description; Do you have a suggestion for that?

**Comment by pmatilai** [comment]
LSB is terribly outdated wrt modern rpm, but is probably the most complete thing there is:
http://refspecs.linuxbase.org/LSB_5.0.0/LSB-Core-generic/LSB-Core-generic/pkgformat.html
